From b5754db0e55882ff063549ee1cac3ec4a9cbfa5d Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Tue, 7 May 2019 18:52:41 +0200
Subject: [PATCH] TRITON-1648 triton-logarchiver shouldn't left workspace
 "dirty" after a successful build Reviewed by: Mike Gerdts
 <mike.gerdts@joyent.com> Approved by: Mike Gerdts <mike.gerdts@joyent.com>

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 39489d6..83de73b 100644
--- a/Makefile
+++ b/Makefile
@@ -50,6 +50,7 @@ include ./deps/eng/tools/mk/Makefile.node_modules.defs
 ROOT		:= $(shell pwd)
 RELEASE_TARBALL	:= $(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR	:= /tmp/$(NAME)-$(STAMP)
+DISTCLEAN_FILES += $(RELEASE_TARBALL)
 
 # triton-origin-multiarch-18.1.0
 BASE_IMAGE_UUID = 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
-- 
2.21.0

